### PR TITLE
Oncoprint redesign

### PR DESCRIPTION
**Pull request by shammamah**
## Description of changes 
* Change colors 
* Move all controls to tabs 
* Use `daq.ToggleSwitch` wherever possible 

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

File: tests/dashbio_demos/app_onco_print.py [hunk 1]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='oncoprint-tab', children=[
+                            html.H4(
+                                "What is OncoPrint?"
+                            ),
+                            html.P(
+                                """
+                                The OncoPrint component is used to view multiple genetic
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                The OncoPrint component is used to view multiple genomic
```
according to https://www.bioconductor.org/packages/release/bioc/vignettes/ComplexHeatmap/inst/doc/s8.oncoprint.html

File: tests/dashbio_demos/app_onco_print.py [hunk 2]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                alteration events through an interactive and zoomable
+                                heatmap. It is a React/Dash port of the popular
+                                oncoPrint() function from the BioConductor R
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                oncoPrint() function from the Bioconductor R
```
according to https://www.bioconductor.org/ (and, apparently, Bioconductor is a *collection* of R packages, the very package providing oncoPrint() being ComplexHeatmap)

File: tests/dashbio_demos/app_onco_print.py [hunk 3]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                package. Under the hood, the rendering is done using
+                                Plotly.js built upon D3. Plotly's interactivity allows
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]: "... done with D3 via Plotly.js" What do you think?

File: tests/dashbio_demos/app_onco_print.py [hunk 4]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                the user to bind clicks and hovers to genetic events,
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                you to bind clicks and hovers to genetic events,
```
as we chose in earlier redesigns... (and not sure it should be 'genetic' or 'genomic')

File: tests/dashbio_demos/app_onco_print.py [hunk 5]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                allowing the user to create complex bioinformatic apps
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                letting you create complex bioinformatics apps
```
(we just used 'allow' earlier in the same sentence)

File: tests/dashbio_demos/app_onco_print.py [hunk 6]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                or workflows that rely on crossfiltering.
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                or workflows that leverage crossfiltering.
```
(not sure, but I would think this way)

File: tests/dashbio_demos/app_onco_print.py [hunk 7]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                                        for ds in DATASETS
+                                    ],
+                                    value='cBioPortalData',
+                                ),
+                            ]),
+                            html.Hr(
+                                className='oncoprint-separator'
+                            ),
+                            html.Div([
+                                html.H4('Hover, click, or event data'),
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                html.H4('Event metadata'),
```
seems to me, at least ('hover' and 'click' being event types, apparently)

File: tests/dashbio_demos/app_onco_print.py [hunk 8]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
+                            html.Div([
+                                html.H4('Colors'),
+                                html.Div(
+                                    children=[
+                                        html.Div(
+                                            className='oncoprint-option-name',
+                                            children='Track color'
+                                        ),
+                                        html.P(
+                                            'Change the default background '
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                            'Change default background '
```

File: tests/dashbio_demos/app_onco_print.py [hunk 9]
```diff
@@ -102,219 +102,206 @@ def header_colors():
...
-                                                    '0': '0',
-                                                    '0.02': '0.02',
-                                                    '0.04': '0.04',
-                                                    '0.06': '0.06',
-                                                    '0.08': '0.08',
-                                                    '0.1': '0.1',
-                                                },
-                                            ),
-                                        ],
+                                        'Adjust the padding (as percentage) '
```
Comment (mkcor) [inline tests/dashbio_demos/app_onco_print.py]:
```suggestion
                                        'Adjust padding (as percentage) '
```

File: tests/dashbio_demos/app_onco_print.py (hunk 3 above)
Comment (shammamah) [reply tests/dashbio_demos/app_onco_print.py]: I think this sounds good!

Comment (mkcor) [reply tests/dashbio_demos/app_onco_print.py]: That's my suggestion, because I find "... done using Plotly.js built upon D3" a bit heavy.

Comment (shammamah) [reply tests/dashbio_demos/app_onco_print.py]: Fixed in dd7f13d!

**Pull request merged**